src/components/GameItem.js

```diff
@@ -4,23 +4,53 @@ import ItemIcons from '../ItemIcons.js';
 import PropTypes from 'prop-types';
 
 class GameItem extends Component {
-  propTypes = {
+  constructor() {
+    super();
+    this.state = {
+      clickStatus: false 
+    }
+
+  }
+  static propTypes = {
     height: PropTypes.number.isRequired,
     layer: PropTypes.number.isRequired,
   }
 
+  onItemSpotted = () => {
+    console.log("item spotted")
+    if (this.props.type === 'litter'){
+      this.props.itemClicked()
+    }
+    this.setState({
+      clickStatus: true,
+    })
+  }
+
   render() {
     const itemStyle = {
       bottom: `${this.props.height}px`, // use props.height to offset from the bottom of screen
       zIndex: this.props.layer, // use props.layer to set z-index, so we display ontop of background
     };
 
     // Update this to select the correct icon for each item
-    const icon = ItemIcons.rock;
+    const icon = ItemIcons[this.props.type]
+
+    let spottedStyling = "game-item"
+
+    if (this.state.clickStatus && this.props.type === 'litter')  {
+      spottedStyling = "game-item spotted-litter"
+    }
+    if (this.state.clickStatus && this.props.type !== 'litter') {
+      spottedStyling = "game-item spotted-nature"
+    }
```

Comment:
Consider any of the following:

1. Turning this into a nested `if` statement
2. Turning this into `if ... else if` (this is my preferred solution
3. instead of re-assigning `spottedStyling` in your conditionals, concatenating it

Reading through your code even inspired me, and in my own solution, I would consider making `spottedStyling` an array of classes that eventually get `join`ed together with a `" "`... hehehehe. But that's just a different refactoring all together.

Also, mind your semicolons! :D